codecov.yml

```diff
@@ -2,6 +2,9 @@ codecov:
     bot: "codecov-io"
     require_ci_to_pass: false
 
+ignore:
+  - "**/*.unit.*"
+
```

Comment:
https://docs.codecov.com/docs/ignoring-paths

Reply:
Should we maybe consider ignoring this [in Jest config](https://github.com/metabase/metabase/blob/04e56d0148d08d03acaa62fd2989eb4e6587b5e7/jest.unit.conf.json#L46), so it doesn't squeeze into coverage no matter what third-party tool we use?

Reply:
It doesn't hurt to add the glob pattern there as well.
But would you then remove it from `codecov`? I don't know enough about the internals of codecov and am not sure whether or not removing this will mess with the calculation.

Reply:
I believed it should be removed from here, once we ignore that in our [jest config](https://github.com/metabase/metabase/blob/04e56d0148d08d03acaa62fd2989eb4e6587b5e7/jest.unit.conf.json#L46-L54), because if there wouldn't be report on the unit tests on the coverage directory, and they shouldn't be included into the report in the first place if we're not going to use them anyway.

Reply:
> I wonder if we should add `modules/` for backend.

Yeah, I saw you ask that on Slack.
Let's hope that @metabase/backend-developers will respond here.

We might have to update the target after that.